### ISSUE TITLE
Error: no member named 'TabBarGetTabName' in namespace 'ImGui'

### ISSUE THREAD

**Issue by jacobfriedman** [ocornut/imgui_test_engine#7]
FYI- Making the latest imgui_test_suite:

```
c++ -I../ -I../../imgui -I../../imgui/backends -I../libs -g -Wall -Wformat -DIMGUI_TEST_ENGINE_ENABLE_IMPLOT=0 -DIMGUI_USER_CONFIG=\"imgui_test_suite/imgui_test_suite_imconfig.h\" -DIMGUI_APP_SDL_GL3 -D_THREAD_SAFE -I/opt/homebrew/include -I/opt/homebrew/include/SDL2 -std=c++14 -c -o imgui_tests_widgets.o imgui_tests_widgets.cpp
imgui_tests_widgets.cpp:1234:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[0]), "Tab 0");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1235:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[1]), "Tab 1");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1236:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[2]), "Tab 2");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1243:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[0]), "Tab 0");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1244:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[1]), "Tab 2");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1249:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[0]), "Tab 0");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1250:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[1]), "Tab 2");
                        ~~~~~~~^
../imgui_test_engine/imgui_te_context.h:574:69: note: expanded from macro 'IM_CHECK_STR_EQ'
#define IM_CHECK_STR_EQ(_LHS, _RHS)                 IM_CHECK_STR_OP(_LHS, _RHS, ==, true, ImGuiTestCheckFlags_None)
                                                                    ^~~~
../imgui_test_engine/imgui_te_context.h:549:100: note: expanded from macro 'IM_CHECK_STR_OP'
        bool __res = ImGuiTestEngine_CheckStrOp(__FILE__, __func__, __LINE__, _FLAGS, #_OP, #_LHS, _LHS, #_RHS, _RHS); \
                                                                                                   ^~~~
imgui_tests_widgets.cpp:1251:32: error: no member named 'TabBarGetTabName' in namespace 'ImGui'
        IM_CHECK_STR_EQ(ImGui::TabBarGetTabName(tab_bar, &tab_bar->Tabs[2]), "Tab 1");
```

**Comment by ocornut** [comment]
Pushed fix 5d3a99a a few hours ago.

**Comment by ocornut** [comment]
Ah, never mind: 
- TestSuite is generally designed to work on latest ImGui.
- But when possible its nice to get it working for older versions.
- Will push a fix for that too.

**Comment by jacobfriedman** [comment]
;) Thanks, just reporting where I can.

**Comment by ocornut** [comment]
My bad- and previous message was incorrect. I thought this was a TestSuite only fix but it did affect TestEngine through the TabBarCompareOrder() helper function. Now pushed fix 748594f

Thank you for reporting!
